app/inc/constants.php

```diff
@@ -28,7 +28,8 @@
 }
 
 // Special modes for the app
-define('DEBUG', (strstr(VERSION, 'dev') || isset($_GET['debug'])) ? true : false);
+define('DEBUG',     strstr(VERSION, 'dev') || isset($_GET['debug']));
+define('LOCAL_DEV', isset($server_config['dev']) && $server_config['dev']);
```

Comment:
we use it in the dev-setup script to install or not a snapshot:
https://github.com/mozfr/transvision/blob/master/app/scripts/dev-setup.sh#L79

Yes, I can add dev=false to -dist

There is a difference between a dev setup and a dev version (in that context it means beta), maybe we should use beta instead but I would prefer that to be part of a separate issue.